### PR TITLE
ServerIron / ServerIron ADX support

### PR DESCRIPTION
**Pull request by xbeaudouin**
Here the ServerIron ADX support for librenms

Regards

### CODE REVIEW

**Comment by laf** [comment]
:+1:  from me.

**Comment by xbeaudouin** [comment]
I just saw I forgot to add the Chassis Temperature sensor :D

**Comment by xbeaudouin** [comment]
ping ?

**Comment by laf** [comment]
@librenms/reviewers

File: includes/discovery/sensors/temperatures/serveriron.inc.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    // Chassis temperature (default)
+    $high_limit		= 110;
+    $high_warn_limit	= 95;
+
+    $descr    = "Chassis Temperature";
+    $oid      = "1.3.6.1.4.1.1991.1.1.1.1.18.0"; // snChasActualTemperature
+    $warn_oid = "1.3.6.1.4.1.1991.1.1.1.1.19.0"; // snChasWarningTemperature
+    $high_oid = "1.3.6.1.4.1.1991.1.1.1.1.20.0"; // snChasShutdownTemperature
+    $value    = snmp_get($device, $oid, '-Oqv', 'FOUNDRY-SN-AGENT-MIB');
```
Comment (murrant) [inline includes/discovery/sensors/temperatures/serveriron.inc.php]: You don't need to specify a mib for files that use raw oids.

Comment (xbeaudouin) [inline includes/discovery/sensors/temperatures/serveriron.inc.php]: I can fix that. Either put the symbolic name or remove the MIB.
What do you mean by "it is done multiple times"?

Regards

**Comment by xbeaudouin** [comment]
@murrant fixed the temperature/sensor. Can you have a look ?

**Comment by Rosiak** [comment]
:+1:

**Comment by paulgear** [comment]
@xbeaudouin Looks like there are some mixed spaces & tabs in includes/discovery/sensors/temperatures/serveriron.inc.php and includes/discovery/os/serveriron.inc.php (that I could see).  Would you mind getting rid of the tabs before we merge?

**Comment by f0o** [comment]
Agree with @paulgear

**Comment by xbeaudouin** [comment]
@paulgear Sure will do RSN.

**Comment by xbeaudouin** [comment]
Done :D

**Pull request merged**